src/Backend/Request.php

```diff
@@ -25,21 +24,6 @@ public function __construct(ClientInterface $client)
     }
 
     /**
-     * @param string $url
-     * @param string $method
-     * @param array  $options
-     *
-     * @return RequestInterface
-     */
-    protected function createRequest($url, $method = 'GET', $options = [])
```

Comment:
This is an internal protected method, not public API. So no breaking change IMO.